bin/batnode-sample.js

```diff
@@ -5,17 +5,18 @@
 const bat_sample = require('commander');
 const chalk = require('chalk');
 
-// const BatNode = require('../batnode').BatNode;
-const BatNode = require('../kad-bat-plugin/batnode.js').BatNode;
 const PERSONAL_DIR = require('../utils/file').PERSONAL_DIR;
 const HOSTED_DIR = require('../utils/file').HOSTED_DIR;
 const fileSystem = require('../utils/file').fileSystem;
 const fs = require('fs');
 
+const BatNode = require('../kad-bat-plugin/batnode.js').BatNode;
+  
 bat_sample
   .description("Demo connection for kad nodes and bat nodes")
   .option('-u, --upload <filePath>', 'upload files from specified file path')
   .option('-d, --download <manifestPath>', 'retrieve files from manifest file path')
+  .option('-a, --audit <manifestPath>', 'audit files from manifest file path')
```

Comment:
When we run a non-local demo we'll want to use the `batchain` command and not `batchain-sample`, so I'm thinking that we should include these changes from `batnode-sample` in `bin/index.js` as well.

Reply:
Noe problem! Updated with new commits.